### DISCUSSION TITLE
Additional Layer of Security for Auth to decrypt keys

### DISCUSSION THREAD

**Discussion by haritvsmurali** [ente-io/ente#2811] [Enhancements]
since both ente photos and auth can be logged in using same account, how about an extra layer of security in auth to decrypt keys? like after logging in,user should have option to input another password to access 2fa codes?

**Comment by ua741** [comment]
We don't have plan to add this additional layer of complexity unless we build more products, and it made sense to add such setting. In the meanwhile, if there's a personal use case, you can create another account for Auth using different email or alias as it's free.
